x-pack/plugins/maps/public/classes/styles/vector/components/legend/symbol_icon.tsx

```diff
@@ -7,13 +7,14 @@
 
 import React, { Component, CSSProperties } from 'react';
 // @ts-expect-error
-import { getMakiSymbolSvg, styleSvg, buildSrcUrl } from '../../symbol_utils';
+import { CUSTOM_ICON_PREFIX_SDF, getSymbolSvg, styleSvg, buildSrcUrl } from '../../symbol_utils';
 
 interface Props {
   symbolId: string;
   fill?: string;
   stroke?: string;
-  style: CSSProperties;
+  style?: CSSProperties;
```

Comment:
Why was this change to `style?: CSSProperties;`?

Reply:
The `style` prop is not used in the [`symbol_utils`](https://github.com/elastic/kibana/blob/main/x-pack/plugins/maps/public/classes/styles/vector/symbol_utils.js#L153-L157) and [`IconSelect`](https://github.com/elastic/kibana/blob/6693ef371f887eca639b09c4c9b15701b4ebabd4/x-pack/plugins/maps/public/classes/styles/vector/components/symbol/icon_select.js#L103-L107) JS modules in main as well as in [`CustomIconsPanel`](https://github.com/elastic/kibana/pull/113144/files#diff-99a7ff9f300a1d94c67213d53f60e0214b3b8dd504630634b8f63e0f4b9e1f8fR158-R162) in this PR. So it seems like it should be an optional prop.